src/OgAccess.php

```diff
@@ -126,7 +126,7 @@ public function userAccess(EntityInterface $group, $operation, AccountInterface
       $permissions = [];
       $user_is_group_admin = FALSE;
 
-      if ($membership = Og::getMembership($user, $group)) {
+      if ($membership = Og::getMembership($group, $user)) {
```

Comment:
For now. I gave a PR that already uses it :)

On Jul 29, 2016 11:57 AM, "Pieter Frenssen" <notifications@github.com>
wrote:

In src/OgAccess.php
<https://github.com/amitaibu/og/pull/273#discussion_r72761062>:

> @@ -126,7 +126,7 @@ public function userAccess(EntityInterface $group, $operation, AccountInterface
>        $permissions = [];
>        $user_is_group_admin = FALSE;
>
> -      if ($membership = Og::getMembership($user, $group)) {
> +      if ($membership = Og::getMembership($group, $user)) {

So we only use this once :)

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/amitaibu/og/pull/273/files/4e93892dc67069bc158e20c8b8d9c862389c734b#r72761062>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AAHrCzL1brACMHbraUlO6YmRsxm2TpwNks5qacBagaJpZM4JXqfM>
.